pkgs/tools/inputmethods/droidmote/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, stdenv, fetchurl }:
+
+let
+  srcs = {
+    x86_64-linux = fetchurl {
+      url = "https://videomap.it/script/dms-ubuntu-x64";
+      sha256 = "1x7pp6k27lr206a8j2pn0wf4wjb0zi28s0g1g3rb08jmr8fh1jnh";
+    };
+    i686-linux = fetchurl {
+      url = "https://videomap.it/script/dms-ubuntu-x32";
+      sha256 = "1d62d7jz50wzk5rqqm3xab66jdzi9i1j6mwxf7r7nsgm6j5zz8r4";
+    };
+    aarch64-linux = fetchurl {
+      url = "https://videomap.it/script/dms-ubuntu-arm64";
+      sha256 = "1l1x7iqbxn6zsh3d37yb5x15qsxlwy3cz8g2g8vnzkgaafw9vva0";
+    };
+
```

Comment:
Can you remove this blank space? My TOC.

Reply:
Thx for the review, done!